mlflow/utils/file_utils.py

```diff
@@ -342,3 +343,28 @@ def _copy_file_or_tree(src, dst, dst_dir=None):
 
 def get_parent_dir(path):
     return os.path.abspath(os.path.join(path, os.pardir))
+
+
+def parse_path(uri):
+    relative_path_uri_prefix = "file:"
+    backslash_count = 1 if os.sep == "/" else 0  # Keep / for linux abs paths
+    fs_prefix_with_localhost = mlflow.tracking.utils._LOCAL_FS_URI_PREFIX[:-1] + "localhost/"
+    if uri.startswith(fs_prefix_with_localhost):
+        path = uri[fs_prefix_with_localhost - backslash_count:]
+    elif uri.startswith(mlflow.tracking.utils._LOCAL_FS_URI_PREFIX):
+        path = uri[len(mlflow.tracking.utils._LOCAL_FS_URI_PREFIX) - backslash_count:]
```

Comment:
I think for absolute path, you might need to explicitly extract the drive on windows? e.g. if have 
```
file_uri = "file:///C/dir/file"
```
This would get me 
```
file_uri = "C/dir/file"
```

which I believe is a relative path?